builder/hcloud/config.go

```diff
@@ -45,6 +46,8 @@ type Config struct {
 	SSHKeys        []string          `mapstructure:"ssh_keys"`
 	Networks       []int64           `mapstructure:"networks"`
 
+	TempSSHKeyLabels map[string]string `mapstructure:"temp_sshkey_labels"`
```

Comment:
I get these are ephemeral labels but I don't know that they need to be named temp since we can just document their usage in the docs. 
```go
	SSHKeyLabels map[string]string `mapstructure:"sshkey_labels"`
```

If prefer to keep them named as temp I would suggest writing them as `temporary_sshkey_labels` to keep consistency with other temporary ssh configuration options. 
```go
	TempSSHKeyLabels map[string]string `mapstructure:"temporary_sshkey_labels"`
```

Reply:
I just thought you'd prefer this ;) I have no opinion about it and can remove the "temp"

Reply:
thanks for looking into this :)